tianshou/policy/modelfree/sac.py

```diff
@@ -63,13 +63,11 @@ def __init__(
         estimation_step: int = 1,
         exploration_noise: Optional[BaseNoise] = None,
         deterministic_eval: bool = True,
-        action_bound_method: str = "tanh",
         **kwargs: Any,
     ) -> None:
         super().__init__(
             None, None, None, None, tau, gamma, exploration_noise,
-            reward_normalization, estimation_step,
-            action_bound_method=action_bound_method, **kwargs)
+            reward_normalization, estimation_step, **kwargs)
```

Comment:
add `action_bound_method="tanh"`? But this will cause DiscreteSACPolicy init error (because it needs `action_bound_method=""`